tmt/steps/provision/__init__.py

```diff
@@ -951,14 +951,17 @@ def _ansible_summary(self, output: Optional[str]) -> None:
                 tasks = fmf.utils.listed(matched.group(1), 'task')
                 self.verbose(key, tasks, 'green')
 
-    def _ansible_playbook_path(self, playbook: Path) -> Path:
+    def _ansible_playbook_path(self, playbook: Path, playbook_root: Path) -> Path:
```

Comment:
I would propose three changes:

* renaming to something like `_sanitize_playbook_path`, and
* making `playbook_root` `Optional[Path]`, and
* call it from `_run_ansible()` for all playbooks, `playbook = self._sanitize_playbook_path(playbook, playbook_root=playbook_root)`. If the root is `None`, method would return the original path, otherwise it would apply its magic.

The idea behind this is that "sanitizing" or "changing" the path argument in some way is now split between two methods: `_ansible_playbook_path()` does the work, and `_run_ansible()` decides when. That makes it harder to maintain: if we add another place that runs Ansible playbooks, we would have to duplicate this logic there as well. On the other hand, if the whole process would be handled by `_ansible_playbook_path()`, all entrypoints would simply call it and let it do the job.

Reply:
Hopefully addressed in 26d311a3